### PR TITLE
simplify win recipe, remove certs recipe (use certifi)

### PR DESCRIPTION
**Pull request by msarahan**
Related: https://github.com/conda-forge/staged-recipes/pull/378

This is currently not building for me locally.  The configure script is picking up the wrong headers for openssl (0.9.8 ???).  Wondering if it's something wrong with my system, or with the recipe.

### CODE REVIEW

**Comment by jakirkham** [comment]
Seems to have conflicts somehow.

File: recipes/curl/build.sh [hunk 1]
```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-export DYLD_LIBRARY_PATH=$PREFIX/lib
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
 
 ./configure \
     --disable-ldap \
+    --with-ca-bundle=$PREFIX/ssl/cacert.pem \
```
Comment (jakirkham) [inline recipes/curl/build.sh]: 👍

File: recipes/curl/meta.yaml [hunk 2]
```diff
@@ -19,10 +20,12 @@ requirements:
   build:
     # python is a build requirement on Windows to resolve features.
     - python    # [win]
+    - pkg-config  # [unix]
     - openssl   # [unix]
     - zlib
   run:
     - openssl   # [unix]
+    - certifi   # [unix]
```
Comment (jakirkham) [inline recipes/curl/meta.yaml:28]: Don't we need it as a build dependency too?

**Comment by jakirkham** [comment]
Yeah, I think you are going to have to merge/rebase with mine.

File: recipes/curl/meta.yaml (hunk 2 above)
Comment (msarahan) [inline recipes/curl/meta.yaml:28]: No.  It is only used at runtime when making connections.

Comment (jakirkham) [inline recipes/curl/meta.yaml:28]: Then how does the line above `--with-ca-bundle=$PREFIX/ssl/cacert.pem` work?

File: recipes/curl/build.sh [hunk 3]
```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-export DYLD_LIBRARY_PATH=$PREFIX/lib
```
Comment (jakirkham) [inline recipes/curl/build.sh:3]: We can't build without this.

Comment (msarahan) [inline recipes/curl/build.sh:3]: I think pkg-config might take its place.

Comment (jakirkham) [inline recipes/curl/build.sh:3]: Nope. I tried merging your stuff local and it didn't work.

File: recipes/curl/meta.yaml (hunk 2 above)
Comment (msarahan) [inline recipes/curl/meta.yaml:28]: It doesn't care that the file doesn't exist.  It just tells it where to look once it goes looking (at runtime)

Comment (jakirkham) [inline recipes/curl/meta.yaml:28]: Ok, thanks for clarifying.

**Comment by jakirkham** [comment]
Thanks @msarahan.

**Pull request merged**